sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -52,6 +58,8 @@
 from apache_beam.utils import proto_utils
 from apache_beam.utils.plugin import BeamPlugin
 
+standard_library.install_aliases()
```

Comment:
Why do we need this in this file?

Reply:
Introduced by futurize step because of `import urllib` ([documentation](http://python-future.org/standard_library_imports.html)). However, since we import urllib from future, this is indeed not required anymore